.github/workflows/nightly.yaml

```diff
@@ -141,6 +141,8 @@ jobs:
     # Still run on test failure
     if: always()
     timeout-minutes: 10
+    env:
+      SLACK_CHANNEL_ID: C04P1V21QF7
```

Comment:
looking at the other examples, i'm pretty sure this can be a human-readable name